.github/workflows/codeflare-sdk-release-sync.yml

```diff
@@ -0,0 +1,92 @@
+---
+# The aim of this GitHub workflow is to update the pipfile to sync with Codeflare-SDK release.
+name: Sync with codeflare-sdk release
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      codeflare_sdk_release_version:
+        required: true
+        description: "Provide the version of the codeflare release you want to update to : "
+      branch:
+        required: true
+        description: "Provide the name of the branch you want to update ex main, vYYYYx etc: "
+
+env:
+  BRANCH_NAME: ${{ github.event.inputs.branch || 'main' }}
```

Comment:
With current setup the `main` branch name is not used.
Either set `main` as `default` in inputs or set `branch` parameter not to be required.

Reply:
Personally I would prefer first option to make it explicit for user what the target branch it.